### PR TITLE
[ad] rename adService to ad

### PR DESCRIPTION
**Pull request by Sozhan308**
# Changes

Renames adservice to ad.

This is part of the ongoing [renaming](https://github.com/open-telemetry/opentelemetry-demo/issues/1788) for standarizing the component names

## Merge Requirements

For new features contributions, please make sure you have completed the following
essential items:

* [x] `CHANGELOG.md` updated to document new feature additions
* [ ] Appropriate documentation updates in the [docs][]
* [ ] Appropriate Helm chart updates in the [helm-charts][]

Maintainers will not merge until the above have been completed. If you're unsure
which docs need to be changed ping the
[@open-telemetry/demo-approvers](https://github.com/orgs/open-telemetry/teams/demo-approvers).

[docs]: https://opentelemetry.io/docs/demo/
[helm-charts]: https://github.com/open-telemetry/opentelemetry-helm-charts

### CODE REVIEW

File: kubernetes/opentelemetry-demo.yaml
Comment (puckpuck) [inline kubernetes/opentelemetry-demo.yaml:1]: This is a generated file; please revert this file.

**Comment by puckpuck** [comment]
Can you add a Changelog entry to this?

File: kubernetes/opentelemetry-demo.yaml
Comment (Sozhan308) [reply kubernetes/opentelemetry-demo.yaml:1]: Sure. I will revert this file

**Comment by Sozhan308** [comment]
> Can you add a Changelog entry to this?

I have updated the changelog :)

**Comment by puckpuck** [comment]
I updated this PR, to update the environment variable `AD_SERVICE_*` -> `AD_*` and the feature flag names.

After the changes, I tested it, and Jaeger displayed traces as expected.

**Comment by Sozhan308** [comment]
> I updated this PR, to update the environment variable `AD_SERVICE_*` -> `AD_*` and the feature flag names.
> 
> After the changes, I tested it, and Jaeger displayed traces as expected.

Thank you for updating and testing the changes @puckpuck. I am new to this project, took me a while to set it up locally to test the changes. Also, would like to know more about the project and if I can contribute to any other topics. Is it okay if I reach out to you @puckpuck, to know more about the project and ongoing topics

File: src/ad/src/main/java/oteldemo/AdService.java [hunk 1]
```diff
@@ -325,4 +325,4 @@ public static void main(String[] args) throws IOException, InterruptedException
     service.start();
     service.blockUntilShutdown();
   }
-}
+}
```
Comment (julianocosta89) [inline src/ad/src/main/java/oteldemo/AdService.java:328]: Add line break

**Review by julianocosta89 — APPROVED** [review]
Added some nits but LGTM

File: src/ad/src/main/java/oteldemo/AdService.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Sozhan308) [reply src/ad/src/main/java/oteldemo/AdService.java]: Updated line break https://github.com/open-telemetry/opentelemetry-demo/pull/1832/commits/6d61d406991294aeb5790611c179e4f1bcc77aab

Comment (Sozhan308) [reply src/ad/src/main/java/oteldemo/AdService.java]: Updated line break https://github.com/open-telemetry/opentelemetry-demo/pull/1832/commits/6d61d406991294aeb5790611c179e4f1bcc77aab

File: kubernetes/opentelemetry-demo.yaml [hunk 2]
```diff
@@ -11023,4 +11023,4 @@ spec:
         volumeMounts:
         - name: config-emptydir
           mountPath: /usr/share/opensearch/config/opensearch.yml
-          subPath: opensearch.yml
+          subPath: opensearch.yml
```
[the remark this answers is not in the document]
Comment (Sozhan308) [reply kubernetes/opentelemetry-demo.yaml]: Updated line break https://github.com/open-telemetry/opentelemetry-demo/pull/1832/commits/6d61d406991294aeb5790611c179e4f1bcc77aab

File: src/ad/gradlew.bat [hunk 3]
```diff
@@ -1,92 +1,92 @@
...
+rem the _cmd.exe /c_ return code!
+set EXIT_CODE=%ERRORLEVEL%
+if %EXIT_CODE% equ 0 set EXIT_CODE=1
+if not ""=="%GRADLE_EXIT_CONSOLE%" exit %EXIT_CODE%
+exit /b %EXIT_CODE%
+
+:mainEnd
+if "%OS%"=="Windows_NT" endlocal
+
+:omega
```
Comment (julianocosta89) [inline src/ad/gradlew.bat:92]: Could you add a line break here as well?

Comment (Sozhan308) [reply src/ad/gradlew.bat]: Yes, added https://github.com/open-telemetry/opentelemetry-demo/pull/1832/commits/2c5397a9ebe00bf5743aadc93db567b3cd2c3800

File: src/ad/gradlew.bat [hunk 4]
```diff
@@ -1,92 +1,92 @@
...
-set EXIT_CODE=%ERRORLEVEL%
-if %EXIT_CODE% equ 0 set EXIT_CODE=1
-if not ""=="%GRADLE_EXIT_CONSOLE%" exit %EXIT_CODE%
-exit /b %EXIT_CODE%
-
-:mainEnd
-if "%OS%"=="Windows_NT" endlocal
-
-:omega
+@rem
```
[the remark this answers is not in the document]
Comment (puckpuck) [reply src/ad/gradlew.bat:1]: I did a hex compare of this file vs the main branch. It's Hex code `0D`, or a Carriage Return added to each line ending. This is PC vs Unix/Mac line endings.

**Pull request merged**